web2/sdk/contracts/erc721-contract.ts

```diff
@@ -0,0 +1,35 @@
+import { write } from "./../utils";
+import { prepareWriteContract, readContract } from "@wagmi/core";
+import { IERC721_ABI } from "../constants";
+
+export class ERC721Contract {
```

Comment:
Lets call this `ERC721Adapter`. Don't forget to rename file as well and code occurrences